NewHorizons/Components/SizeControllers/CometTailController.cs

```diff
@@ -39,8 +39,9 @@ public override void FixedUpdate()
 
         private void UpdateTargetPositions()
         {
-            var toPrimary = (_body.transform.position - _dustTargetBody.transform.position).normalized;
-            var velocityDirection = (_primaryBody?.GetVelocity() ?? Vector3.zero) -_body.GetVelocity(); // Accept that this is flipped ok
+            // body is null for proxies
```

Comment:
This just means proxies will just look broken. Should make it so that the proxy comet tail links to the real comet tail and just copies its rotations. But this isn't related to triplanar so it's fine for this PR if not I guess.